components/Card/package.json

```diff
@@ -16,7 +16,12 @@
       "import": "./dist/mjs/index.js",
       "default": "./dist/cjs/index.js"
     },
-    "./index.css": "./dist/index.css"
+    "./index.css": "./dist/index.css",
+    "./dist/css.mjs": {
+      "types": "./dist/mjs/css.mjs.d.ts",
+      "import": "./dist/mjs/css.mjs",
+      "default": "./dist/mjs/css.mjs"
+    }
```

Comment:
Ik zie dat ik zelf inconsistent ben geweest, maar kan het zijn dat `.mts` de meest correcte file extension is voor bij `.mjs`?